web-server/v0.4/e2etests

```diff
@@ -12,16 +17,22 @@ export default {
   },
 };
 EOF
+# Make sure the yarn environment is all up-to-date, or perform the
+# install ahead of actually running the tests.
+printf -- "Running \"yarn install\" command ...\n" 
```

Comment:
Should the next line be `yarn install  --network-timeout ...`?

Reply:
Yes, I can add `install` as that is the default behavior.

Reply:
Fixed.